app/components/application_component.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
```

Comment:
```suggestion

```

We agreed in an earlier PR that we don't need the frozen_string_literal comment at the top of each class.